.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,40 @@
+name: goreleaser
```

Comment:
Since this repo will only contain a Go library, I don't think we need this file because we don't need GoReleaser to create pre-built binaries.

Reply:
Is go releaser just for binaries? I thought it was triggered whenever releases were made

Reply:
GoReleaser can do other things but we only use it for creating binaries in celestia-app